awscli/examples/kafka/create-configuration.rst

```diff
@@ -6,7 +6,7 @@ The following ``create-configuration`` example creates a custom MSK configuratio
         --name "CustomConfiguration" \
         --description "Topic autocreation enabled; Apache ZooKeeper timeout 2000 ms; Log rolling 604800000 ms." \
         --kafka-versions "2.2.1" \
-        --server-properties file://configuration.txt
+        --server-properties fileb://configuration.txt
```

Comment:
This change should be specific to the CLI v2 right? Do we need it on that branch specifically? We may want to reach out to @elysahall to see if there's a strategy for managing doc variations between the two branches. Otherwise, this may regress with future updates.

Reply:
This topic has come up before, last I knew of there is no strategy for having separate V1 and V2 examples. I'll touch base with the team to see if this is still the case.

Reply:
I think we should just merge it into the v2 branch. Potentially this will cause issues when merging the v1 branch back into v2 during updates. Someone would need to deal with that merge conflict manually.

Reply:
Ok I just changed the branch to v2.